documentation/release_notes.markdown

```diff
@@ -61,179 +94,253 @@ This release retires all api versions before v4 and promotes v4 to be
+    `puppetlabs.puppetdb` namespace. Any monitoring applications you have that
+    expect the old namespace will need to be changed.
 
-* Expanded data format is now more formalized. ([PDB-1228](https://tickets.puppetlabs.com/browse/PDB-1228))
+* The endpoints `aggregate-event-counts` and `event-counts` have now been marked as experimental.
 
-    In the past in the experimental /v4 endpoint we used to return related data for endpoints directly
-    in the JSON response with very little formality. However, due to internal changes that only work for
-    PostgreSQL we've decided to formalize the format of returning expanded/related data so that some of
-    it may be marked as optional from a contractual perspective.
+    These endpoints were introduced to solve specific problems for the PE console,
+    however the design is under question. We're moving these endpoints back to
+    experimental status so that we are able to deliver a better option later on.
 
-    In the past for example, report events would return an array of events, now it returns that data inside
-    the `data` key, and a link to the original data in a `href` key.
+* The older puppetdb-\* sub-commands are now retired (`puppetdb-ssl-setup` for example) instead,
+  the new sub-command style should be used `puppetdb ssl-setup`. ([PDB_663](https://tickets.puppetlabs.com/browse/PDB-663))
 
-    Note the `data` key is only supported on PostgreSQL, users of HSQLDB will be have to retrieve the related
-    data via the `href` URL in the response. Any support for expanding related data in the past for HSQLDB
-    has now been dropped in favor of less complexity and the better performance focus around PostgreSQL.
+* Retire support for PostgreSQL versions less than 9.4 ([PDB-1592](https://tickets.puppetlabs.com/browse/PDB-1592))
 
-    An example expanded block would look something like this:
+* Retire support for RedHat 5, Debian 6 & Ubuntu 10.04 ([PDB-663](https://tickets.puppetlabs.com/browse/PDB-663))
 
-        {
-          "href": "/pdb/query/v4/reports/32c821673e647b0650717db467abc51d9949fd9a/events",
-          "data": [ ... ]
-        }
+* Retire `[global]` configuration key `event-query-limit` ([PDB-693](https://tickets.puppetlabs.com/browse/PDB-693))
 
-    For API consumers this means, the extra `data` key needs to be traversed to access the real related
-    object data.
+* Retire `[database]` configuration key `node-ttl-days` ([PDB-644](https://tickets.puppetlabs.com/browse/PDB-644))
 
-* Add "noop" flag to reports ([PDB-1177](https://tickets.puppetlabs.com/browse/PDB-1177))
+* Retire `[jetty]` configuration key `certificate-whitelist` ([PDB-886](https://tickets.puppetlabs.com/browse/PDB-886))
 
-    This adds a "noop" field to the reports object, which is a
-    required boolean flag indicating whether the run producing the
-    report was a --noop.
+    This now lives in the `[puppetdb]` section.
```

Comment:
Move `[jetty]` configuration key `certificate-whitelist` to the `[puppetdb]` section.